examples/mini-app/src/hooks/useGameHistory.ts

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect, useCallback } from "react"
+import {
+  fetchBets,
+  Bet_OrderBy,
+  OrderDirection,
+  CasinoBet,
+  CasinoChainId,
+  formatRawAmount,
+  FORMAT_TYPE,
+  CASINO_GAME_TYPE,
+} from "@betswirl/sdk-core"
+import { useAccount } from "wagmi"
+import { TokenImage } from "@coinbase/onchainkit/token"
+import { ETH_TOKEN } from "../lib/tokens"
+import { createLogger } from "../lib/logger"
+
+const logger = createLogger("useGameHistory")
+
+enum HistoryEntryStatus {
+  WonBet = "Won bet",
+  Busted = "Busted",
+}
+
+export interface HistoryEntry {
+  id: string
+  status: HistoryEntryStatus
+  multiplier: number | string
+  payoutAmount: number | string
+  payoutCurrencyIcon: React.ReactElement
+  timestamp: string
+}
+
+function formatRelativeTime(timestampSecs: number): string {
+  const now = new Date()
+  const then = new Date(timestampSecs * 1000)
+  const diffInSeconds = Math.floor((now.getTime() - then.getTime()) / 1000)
+
+  if (diffInSeconds < 0) return "in the future"
+  if (diffInSeconds < 5) return "just now"
+  if (diffInSeconds < 60) return `~${diffInSeconds}s ago`
+
+  const diffInMinutes = Math.floor(diffInSeconds / 60)
+  if (diffInMinutes < 60) return `~${diffInMinutes}m ago`
+
+  const diffInHours = Math.floor(diffInMinutes / 60)
+  if (diffInHours < 24) return `~${diffInHours}h ago`
+
+  const diffInDays = Math.floor(diffInHours / 24)
+  return `~${diffInDays}d ago`
+}
+
+export const useGameHistory = (
+  gameType: CASINO_GAME_TYPE,
+  userAddress?: `0x${string}`,
```

Comment:
will any other user's history ever be displayed, do we really need the arg or can we just take it from useAccount()
?